### DISCUSSION TITLE
NSE_FO

### DISCUSSION THREAD

**Discussion by vu2rps** [pparesh25/Getbhavcopy-alternative#11] [General]
Sir this too needs to be modified I suppose unless I'm doing something wrong. Or maybe you be 
posting the modified in days to come, Thanks for modifying the equity.

**Comment by pparesh25** [comment]
Added py file >>> New_Nse_fo_roman_suffixes.py <<< for symbol name with roman suffixes
Also added >>> NSE_EOD_FO.zip <<< file for historical fo_eod files with symbol name with roman suffixes to fill gap in database
